src/app/screens/books/components/book/book.component.scss

```diff
@@ -0,0 +1,27 @@
+.book {
+  box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+  max-width: 300px;
+  min-width: 200px;
+  padding: 1rem;
+  position: relative;
+  transition: all 0.3s cubic-bezier(.25,.8,.25,1);
```

Comment:
is all value necessary?

Reply:
I use "all" to make the transition more flexible apply to all properties.

Do you think that is necessary to apply to an exact property?